tools/dep_updaters/update-ada.sh

```diff
@@ -11,7 +11,11 @@ DEPS_DIR="$BASE_DIR/deps"
 . "$BASE_DIR/tools/dep_updaters/utils.sh"
 
 NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
-const res = await fetch('https://api.github.com/repos/ada-url/ada/releases/latest');
+const res = await fetch('https://api.github.com/repos/ada-url/ada/releases/latest', {
```

Comment:
Maybe we can have something like that, so scripts remains manually  executable outside gh actions?
```suggestion
const res = await fetch('https://api.github.com/repos/ada-url/ada/releases/latest', process.env.GITHUB_TOKEN && {
```

Reply:
It can already be executed manually even if you don't have a GITHUB_TOKEN env, simply the request will be un-authenticated

Reply:
Indeed, I thought that the request would be rejected by gh if the token was invalid (`Bearer undefined`). That's good to know.